JSIL.Libraries/Sources/JSIL.Core.js

```diff
@@ -5635,6 +5635,10 @@ JSIL.MakeType = function (typeArgs, initializer) {
       };
     } else {
       typeObject._IsAssignableFrom = function (typeOfValue) {
+        if (!typeOfValue.__TypeInitialized__ && true) {
```

Comment:
No need to change it, just wondering if anything special was going on there. Thanks.